src/scss/lr7-classic-theme/_list-group.scss

```diff
@@ -54,6 +54,7 @@
 // Tabular List Group
 
 .tabular-list-group {
+	@include box-shadow(0 1px 4px 0 rgba(0,0,0,0.15));
```

Comment:
Box-shadow mixin not needed. Ie9 supports box-shadow without browser prefix. We are aiming for ie9+. This also applies to border-radius.